### PR TITLE
update faqs

### PR DESCRIPTION
**Pull request by MichelDiz**
Related to #2940, #2941 and #2942

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/2943)

### CODE REVIEW

**Comment by danielmai** [comment]
___
*[wiki/content/faq/index.md, line 49 at r1](https://reviewable.io/reviews/dgraph-io/dgraph/2943#-LXLR1EL0Mw0H22GJ4Qv:-LXLR1EM0JLCQqk5T-LZ:b-841ayj) ([raw file](https://github.com/dgraph-io/dgraph/blob/d22e9112343d7038cc34c8005259ad4cea049620/wiki/content/faq/index.md#L49)):*
Suggested revision:
> Today we use [Badger](https://github.com/dgraph-io/badger), an efficient and persistent key-value database we built that's written in Go. Our blog covers our rationale for choosing Badger in ["Why we choose Badger over RocksDB in Dgraph"](https://blog.dgraph.io/post/badger-over-rocksdb-in-dgraph/). Today, Badger is used in Dgraph as well as [many other projects](https://github.com/dgraph-io/badger#other-projects-using-badger).

This adds a link both to the Badger repo and the recent blog post. There's no need to link to the original blog post when there's the newer one (which links to the original post already), both of which cover what Badger is and why we use it. It's clearer to state what Badger is today (a database) and not what it was in the beginning (a store). There's a bunch more features that modern Badger has compared to what it had at the beginning.

File: wiki/content/faq/index.md [hunk 1]
```diff
@@ -43,8 +43,10 @@ Dgraph v0.8 and above uses [Badger](https://github.com/dgraph-io/badger), a pers
...
+### Why doesn't Dgraph use BoltDB or RocksDB?
+BoltDB depends on a single global <code>RWMutex</code> lock for all reads and writes; this negatively affects concurrency of iteration and modification of posting lists for Dgraph. For this reason, we decided at that time not to use it and instead use RocksDB. On the other hand, RocksDB supports concurrent writes and is being used in production both at Google and Facebook.
+
+Later middle 2017, We've decided to built an efficient and persistent log structured merge (LSM) tree based key-value store, purely in Go language, for Dgraph. We called it Badger. We have a Blog post covering the whole motives to built Badger in ["Introducing Badger: A fast key-value store written purely in Go"](https://blog.dgraph.io/post/badger/). Today (2019) not only Dgraph uses Badger, as you can see several others use it too. See [Projects Using Badger](https://github.com/dgraph-io/badger#other-projects-using-badger)
```
Comment (marigonzes) [inline wiki/content/faq/index.md]: s/built/build

**Comment by marigonzes** [comment]
I was pointing to you there was a typo in "built", where it should have been "build". But in this new version, that problem is no longer present.

**Comment by danielmai** [comment]
Thanks @MichelDiz for updating these sections and @marigonzes for reporting and reviewing.

**Pull request merged**